internal/db/schema/migrations/oss/postgres_56_01_test.go

```diff
@@ -2,26 +2,23 @@ package oss_test
 
 import (
 	"context"
-	"crypto/rand"
 	"testing"
 
-	"github.com/hashicorp/boundary/internal/auth/oidc"
 	"github.com/hashicorp/boundary/internal/auth/password"
+	"github.com/hashicorp/boundary/internal/authtoken"
 	"github.com/hashicorp/boundary/internal/db"
 	"github.com/hashicorp/boundary/internal/db/common"
 	"github.com/hashicorp/boundary/internal/db/schema"
-	"github.com/hashicorp/boundary/internal/iam"
 	"github.com/hashicorp/boundary/internal/kms"
 	"github.com/hashicorp/boundary/internal/types/scope"
 	"github.com/hashicorp/boundary/testing/dbtest"
-	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
-func TestMigrations_DeleteOrphanedAccounts(t *testing.T) {
+func TestMigrations_NewForeignKeys(t *testing.T) {
 	const (
-		priorMigration   = 55001
-		currentMigration = 56001
+		priorMigration   = 56001
+		currentMigration = 56002
```

Comment:
Excuse my lack of understanding here, but why wouldn't this be `55001 -> 56001`?

Reply:
Discussed on call, this is correct.